pom.xml

```diff
@@ -63,6 +63,7 @@
         <artifactId>google-cloud-errorreporting</artifactId>
```

Comment:
should artifact id separate "error" from "reporting" with dash?

Reply:
Historically, this repo has both errorreporting and error-reporting mixed everywhere. The gapic module is named `google-cloud-errorreporting` and the proto and grpc modules have `proto-google-cloud-error-reporting-v1beta1`  and `grpc-google-cloud-error-reporting-v1beta1` respectively. The artifact Ids published on maven central also reflect this.

The purpose of this PR is to migrate the existing repo as-is to owlbot without any breaking changes. Any changes in the module names or artifact IDs will be breaking changes and out of scope for his PR.